teamcode.html

```diff
@@ -61,8 +61,10 @@ <h2>TeamCode</h2>
       <p>The TeamCode team for Fall 2019 consists of:</p>
       <ul>
         <li><b>Brenton Strine</b> (Advisor)</li>
-        <li><b>Mustafa Alsaeedi</b> (Developer)</li>
+        <li><b>Mustafa Alsaeedi</b> (Devloper)</li>
```

Comment:
Hey @team-refcode/teamcode it looks like you are introducing a regression. 

A regression is when you undo a fix that someone else did. @24sint had corrected the typo here ("Devloper" should be spelled "Developer") in her code. However, your branch is built off of older code, before she made that fix, so you are reintroducing the "Devloper" typo.

Reply:
ok. i will fix it. i did not realize that. thank you